tokio/src/util/linked_list.rs

```diff
@@ -29,6 +30,7 @@ pub(crate) struct LinkedList<L, T> {
 
 unsafe impl<L: Link> Send for LinkedList<L, L::Target> where L::Target: Send {}
 unsafe impl<L: Link> Sync for LinkedList<L, L::Target> where L::Target: Sync {}
+impl<L: Link> UnwindSafe for LinkedList<L, L::Target> {}
```

Comment:
Since the linked list is just raw pointers, having manual impls for both traits is probably the right call.